tests/unit/test_class_cmds.py

```diff
@@ -1269,9 +1269,9 @@ class TST_MemberOfFamilyCollection {
 
     ['Verify class command invokemethod fails Invalid Method',
      ['invokemethod', 'CIM_Foo', 'Fuzzyx', '-p', 'TestInOutParameter=blah'],
-     {'stderr': ["CIMError: 17"],
+     {'stderr': ["Error: ClickException: .* CIM_Foo .* Fuzzyx"],
```

Comment:
I was trying to leave the Error: off the stdout test since that comes from click and they can change on us.

Reply:
We have the "Error:" string in many (37?) expected message string definitions in our tests. Do we want to remove it from all of them?

Reply:
Leave as is